pure-docker/deploy.sh

```diff
@@ -49,6 +48,5 @@ do
 done
 
 for i in $(seq 0 $(($NUM_FRONTEND - 1))); do ./deploy-frontend.sh $i; done
-# Not used in customer-replica branch.
-#./deploy-caddy.sh
+./deploy-caddy.sh
```

Comment:
I think the changes in this file need to not be included as well

Reply:
Ah yikes - thanks!